.github/workflows/update-nimbus-experiments.yml

```diff
@@ -0,0 +1,36 @@
+name: "Update Nimbus Experiments"
+
+on:
+  schedule:
+    - cron: '*/30 * * * *'
+
+jobs:
+  update-nimbus-experiments:
+    name: "Update Nimbus Experiments"
+    runs-on: ubuntu-20.04
```

Comment:
any particular reason for using specific ubuntu instance, all other workflows are using `ubuntu-latest` 🤔

Reply:
Nope! It was based off a different repo's workflows, I'll update it to latest!

Reply:
I've updated the runner to be `ubuntu-latest`